scripts/move_session_workspace.py

```
@@ -0,0 +1,667 @@
+    return p
+
+
+def _pg_dump(out_path: str) -> None:
+    import subprocess
+
+    from src.config import settings
+
+    uri = settings.DB.CONNECTION_URI
+    # strip the +psycopg driver suffix for libpq / pg_dump
+    libpq = uri.replace("postgresql+psycopg", "postgresql")
+    subprocess.run(["pg_dump", "--dbname", libpq, "--file", out_path], check=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)